examples/bsc-p2p/src/block_import/mod.rs

```diff
@@ -0,0 +1,47 @@
+use reth_engine_primitives::EngineTypes;
+use reth_network::import::BlockImport;
+use reth_network_peers::PeerId;
+use reth_payload_primitives::BuiltPayload;
+use reth_primitives::NodePrimitives;
+use service::{BlockMsg, ImportHandle, Outcome};
+use std::{
+    fmt,
+    task::{ready, Context, Poll},
+};
+
+pub mod service;
+
+pub struct BscBlockImport<EngineT: EngineTypes> {
+    handle: ImportHandle<EngineT>,
+}
```

Comment:
we can keep this wrapper type for now but maybe we can also impl the trait directly on the handle